test/dotnet.Tests/CommandTests/BuildServerShutdownCommandTests.cs

```diff
@@ -157,6 +160,32 @@ public void GivenAFailingShutdownItPrintsFailureMessage()
             VerifyShutdownCalls(mocks);
         }
 
+        [Fact]
+        public void GivenARunningRazorServerItShutsDownSuccessfully()
+        {
+            var testInstance = TestAssets.Get("TestRazorApp")
+                .CreateInstance()
+                .WithSourceFiles();
+
+            new BuildCommand()
+                .WithWorkingDirectory(testInstance.Root)
+                .Execute()
+                .Should()
+                .Pass();
+
+            new TestBuildServerCommand()
+                .WithWorkingDirectory(testInstance.Root)
+                .ExecuteWithCapturedOutput("shutdown --razor")
+                .Should()
+                .Pass()
+                .And
+                .HaveStdOutMatching(
+                    string.Format(
+                        LocalizableStrings.ShutDownSucceededWithPid,
+                        CommandLocalizableStrings.RazorServer,
+                        @"\d+").Replace("(", @"\(").Replace(")", @"\)"));
```

Comment:
nit:what is this? looks flaky

Reply:
The message contains `(process ####)`, so this turns that part into `\(\d+\)` to match a regex.

This is being removed anyway with my next push to ensure this test properly isolates the running Razor server (both with the pipe name it uses and the pid directory it creates the pid file in).

Reply:
I've pushed up that change.